app/models/review.rb

```diff
@@ -0,0 +1,24 @@
+class Review < ApplicationRecord
+  belongs_to :park
+  belongs_to :user
+
+  validates :park_rating,
+    numericality: { greater_than_or_equal_to: 1, less_than_or_equal_to: 5 },
+    allow_nil: true
+  validates :dog_friendly_rating,
+    numericality: { greater_than_or_equal_to: 1, less_than_or_equal_to: 5 },
+    allow_nil: true
+  validates :camping_rating,
+    numericality: { greater_than_or_equal_to: 1, less_than_or_equal_to: 5 },
+    allow_nil: true
+  validates :playground_rating,
+    numericality: { greater_than_or_equal_to: 1, less_than_or_equal_to: 5 },
+    allow_nil: true
+  validates :hiking_rating,
+    numericality: { greater_than_or_equal_to: 1, less_than_or_equal_to: 5 },
+    allow_nil: true
+  validates :scenery_rating,
+    numericality: { greater_than_or_equal_to: 1, less_than_or_equal_to: 5 },
+    allow_nil: true
+
+end
```

Comment:
There is probably a more DRY way to write this, but since there's not a lot else going on in this model, I don't mind the clear-cut nature of this approach.